pkg/build/generator/generator_test.go

```diff
@@ -741,7 +741,7 @@ func TestGenerateBuildFromConfig(t *testing.T) {
 	bc := &buildapi.BuildConfig{
 		ObjectMeta: kapi.ObjectMeta{
 			Name:      "test-build-config",
-			Namespace: "test-namespace",
+			Namespace: kapi.NamespaceDefault,
```

Comment:
(I don't *think* this particular test actually cares what the namespace value is, but it seems odd to change it)